scripts/pdf/render.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+#A Python script to generate a single PDF document with all the tldr pages. It works by generating 
+#intermediate HTML files from existing md files using Python-markdown, applying desired formating 
+#through CSS, and finally rendering them as PDF. There is no LaTeX dependency for generating the PDF.
+
+import os
+import sys
+import glob
+import re
+import markdown
+import argparse
+
+from weasyprint import HTML
+
+def main(loc, colorscheme):
+
+    oslist = []
+    allmd = []
+    group = []
+    ap = []
+
+    #Checking correctness of path
+    if not os.path.isdir(loc):
+        print("Invalid directory. Please try again!", file = sys.stderr)
+        sys.exit(1)
+
+    if not loc.endswith('/'):
```

Comment:
I don't think this is required. It works fine without the trailing slash. See my comment below.